cli/js/event_target_test.ts

```diff
@@ -34,11 +37,19 @@ test(function constructedEventTargetCanBeUsedAsExpected(): void {
 
 test(function anEventTargetCanBeSubclassed(): void {
   class NicerEventTarget extends EventTarget {
-    on(type, callback?, options?): void {
+    on(
+      type: string,
+      callback: (e: Event) => void,
```

Comment:
This was potion before the types.

Reply:
I'm sorry but I'm not sure what you mean

Reply:
Same issue as https://github.com/denoland/deno/pull/3899#discussion_r377594434

Reply:
Right so actually I guess it should be typed `(e: Event) => void | null` which is what TypeScript now expects with `strictNullChecks` and would also match TypeScript's DOM types.

Reply:
Right so actually I guess it should be typed `(e: Event) => void | null` which is what TypeScript now expects with `strictNullChecks` and would also match TypeScript's DOM types.